### ISSUE TITLE
Problem with FitsException and and Stil library

### ISSUE THREAD

**Issue by JonCook** [nom-tam-fits/nom-tam-fits#78]
Hi,

I've seen recently that FitsException has moved into a new common package. From a backwards compatible perspective this isn't ideal but I've updated our code to use this however I now have a problem with a different external library. 

We use the Stil library http://www.star.bristol.ac.uk/~mbt/stil/ which internally references nom.tam.fits but after the FitsException move, this library can no longer resolve the exception so I get a runtime exception from this library. 

In short this change breaks the Stil library and is a show stopper for us upgrading from 1.06. Is there any possibility to put back the exception into its old place as perhaps deprecated and still have the new one in common? At least this way you ensure you don't break any clients.

Thanks
Jon

**Comment by ritchieGitHub** [comment]
That should not have happened. Sorry!
Erik can you change the package back?

**Comment by erikfk** [comment]
Done.Sorry. I was too concentrated on removing cyclic dependencies - I guess we can live with them a little bit longer ;-)

**Comment by ritchieGitHub** [comment]
@JonCook could you try again?

**Comment by JonCook** [comment]
sure. It doesn't look like the snapshot is available yet? https://oss.sonatype.org/content/repositories/snapshots/gov/nasa/gsfc/heasarc/nom-tam-fits/1.14.1-SNAPSHOT/

Thanks

**Comment by JonCook** [comment]
Yes all good. Thanks. Phew!
